yt/frontends/stream/tests/test_stream_stretched.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+from yt import load_uniform_grid
+from yt.testing import assert_almost_equal, assert_equal
+
+
+def test_variable_dx():
```

Comment:
Stakeholders that know of are almost exclusively using non-cartesian geometries, so I'd say "probably"

Reply:
So I'm not sure.  I looked at it and I think it's very straightforward to add if we use `pytest.parameterize` (can we do that now?) but I also don't know that we actually gain any info.

Reply:
So if we do this, we'll need to change our axis name index field checks -- which I'm actually not sure gains us any information.  One option instead would be to do a check on the total volume.  But again, I'm kinda not sure yet if it's worth it.

Reply:
We can use pytest.mark.parametrize but only at the cost of making the test module pytest-only (which means adding it to *two* blacklists) I'd say it's not necessarily worth it here. yt_idefix will be ground 0 for catching any bugs with spherical, cylindrical, and polar geometries ;-)

Reply:
> which I'm actually not sure gains us any information.

Ya, that's why I was uncertain -- I couldn't tell if it'd be more of a test of yt's existing coordinate-system handling or a test of the actual dds implementation. 

> yt_idefix will be ground 0 for catching .... 

I think leaving this test alone and testing dds with different geometries more indirectly through frontends is totally reasonable -- I would like to add some actual image tests for the geographic geometries (if I can figure out cartopy testing...), so that would cover this for the cases I've been playing with.